vue/src/store/modules/workspaces/article/actions.js

```diff
@@ -163,3 +173,161 @@ export function prepareData(data) {
   }
   return data;
 }
+
+/** Article Files Actions */
+
+/**
+ * Get attachment config data
+ *
+ * @param commit
+ * @returns {Promise<void>}
+ */
+export async function getAttachConfig({commit}) {
+  const {success, body} = await httpService.get(`${fileUrl}/get-attach-config`)
+  if (success) {
+    commit(GET_ATTACH_CONFIG, body)
+  }
+}
+
+/**
+ * Upload files
+ *
+ * @param dispatch
+ * @param payload
+ * @param data
+ * @param config
+ * @returns {Promise<unknown>}
+ */
+export async function attachFiles({dispatch}, {data, config}) {
+  const res = await httpService.post(`${fileUrl}/attach-files`, prepareFiles(data), config)
+  if (res.success) {
+    dispatch('getFilesByArticle', data.article_id);
+  }
+  return res;
+}
+
+/**
+ * Get article files data by article
+ *
+ * @param commit
+ * @param articleId
+ * @returns {Promise<void>}
+ */
+export async function getFilesByArticle({commit}, articleId) {
+  const {success, body} = await httpService.get(`${fileUrl}?articleId=${articleId}&expand=updatedBy&sort=name`);
```

Comment:
@skavt You can write get params easier with http service instead of directly in string. Try using this approach everywhere when you need get paras with httpService: 
```
httpService.get(fileUrl, {
      params: {
        articleId,
        expand: 'updatedBy',
        sort: 'name'
      }
    });
```